### PR TITLE
Minor punctuation correction in README.md

### PR DESCRIPTION
**Pull request by Bert-R**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ for more information.
 
 Notary aims to make the internet more secure by making it easy for people to
 publish and verify content. We often rely on TLS to secure our communications
-with a web server which is inherently flawed, as any compromise of the server
+with a web server, which is inherently flawed, as any compromise of the server
```
Comment (eiais) [inline README.md]: I think in this case `which is inherently flawed` is a essential clause and doesn't need the comma.

Comment (HuKeping) [reply README.md]: Hi there, thanks for contributing to the docs!

I'm not a native speaker and I can hardly tell which is the right one.

What we want to tell people is, the old-ways(only rely on TLS) to secure our communication is inherently flawed.

Comment (Bert-R) [reply README.md]: Indeed. To communicate that clearly and correctly, the comma would be necessary.

Comment (HuKeping) [reply README.md]: LGTM once Gordon stop shouting

**Comment by endophage** [comment]
@Bert-R it looks like you GPG signed one of the commits. That's not what Gordon is asking for. Please follow the instructions referenced by https://github.com/moby/moby/blob/master/CONTRIBUTING.md#sign-your-work 

You will need to add the "Signed off by" line to all commits in the PR so feel free to squash your 2 commits into 1.

**Comment by Bert-R** [comment]
@endophage I'm sorry to say, but I'm not familiar enough with Git to know how to do that. Can you tell me how?

**Comment by eiais** [comment]
@Bert-R it's a lowercase s flag.
` git commit --amend -s --no-edit`

**Comment by Bert-R** [comment]
That's not the issue, my last commit has the signature, but David asked to squash all commits (so also the merge-ones)

**Comment by eiais** [comment]
Sorry I had edited my comment after posting it.

**Comment by Bert-R** [comment]
Done

**Comment by cyli** [comment]
@Bert-R Thank you so much for this change - apologies but there seem to be 9 commits in this PR now, I think possibly because master got merged in and some other issues.  Would you mind squashing them down to 1?  Alternately, if you wouldn't mind allowing me write access to that branch (possibly that is the default on github now for PRs?) I would be happy to do that rather than make you jump through more hoops.

**Comment by Bert-R** [comment]
@cyli Please go ahead. I've spend a tremendous amount of time on this single comma and leave it up to you guys. If you want to close the PR and add the comma in another one, that would be equally fine with me.

**Comment by HuKeping** [comment]
Help to make a change on it, merging.

**Comment by cyli** [comment]
Thanks @HuKeping.  Sorry I didn’t get around to this.  LGTM

**Pull request merged**